### PR TITLE
NOISSUE - Fix building UI docker image for arm

### PR DESCRIPTION
**Pull request by blokovi**
Add entrypoint in arm Dockerfile
Switch from ash to dash in entrypoint script

Signed-off-by: Ivan Milošević <iva@blokovi.com>

### What does this do?
Fix building of UI docker image for arm architecture

### Which issue(s) does this PR fix/relate to?
No issue

### CODE REVIEW

**Review by nmarcetic — DISMISSED** [review]

**Review by manuio — DISMISSED** [review]

**Review by nmarcetic — DISMISSED** [review]

**Review by mteodor — DISMISSED** [review]

**Review by drasko — DISMISSED** [review]
I will approve, but this should be simplified. I let @nmarcetic do the research and correct this implementation.

**Review by nmarcetic — DISMISSED** [review]

File: ui/docker/Dockerfile.arm [hunk 1]
```diff
@@ -22,3 +22,6 @@ COPY --from=builder /app/main.js /usr/share/nginx/html
 COPY --from=builder /app/css/mainflux.css /usr/share/nginx/html/css/
 COPY --from=builder /app/src/Websocket.js /usr/share/nginx/html/src/
 COPY --from=builder /app/docker/nginx.conf /etc/nginx/conf.d/default.conf
+COPY docker/entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
```
Comment (manuio) [inline ui/docker/Dockerfile.arm]: maybe a new line here?

**Review by manuio — CHANGES_REQUESTED** [review]

File: ui/docker/Dockerfile.arm (hunk 1 above)
Comment (blokovi) [reply ui/docker/Dockerfile.arm]: added

**Review by manuio — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**